examples/device/ep/csrc/cuda_stream.cpp

```
@@ -0,0 +1,43 @@
+    return static_cast<cudaStream_t>(stream);
+}
+
+[[nodiscard]] cudaStream_t
+get_current() {
+    void *stream;
+    TORCH_ERROR_CODE_CHECK(aoti_torch_get_current_cuda_stream(-1, &stream));
+    return static_cast<cudaStream_t>(stream);
+}
+
+void
+set_current(cudaStream_t stream) {
```

Cppcheck (2.21.0)

[style] The function 'set_current' is never used.

(unusedFunction)